### PR TITLE
Syntax Highlight

### PR DESCRIPTION
**Pull request by coderhs**
Add syntax highlight to code2pdf in reference to issue #5 

This PR is incomplete as due to the addition of syntax highlighting the hash of pdf is changing with each render so need to write a test to accommodate that.

### CODE REVIEW

File: spec/code2pdf/convert_to_pdf_spec.rb [hunk 1]
```diff
@@ -32,4 +33,29 @@
...
+      expect(@pdf.send(:prepare_line_breaks, test_text)).to eq('test<br>test')
+    end
+  end
+
+  describe '#syntax_highlight' do
+    before do
+      @pdf = ConvertToPDF.new from: 'spec/fixtures/hello_world',
+                                                    to: 'spec/fixtures/hello_world.pdf'
+    end
+    it 'return file with syntax_highlight html clases' do
```
Comment (lucascaton) [inline spec/code2pdf/convert_to_pdf_spec.rb]: Could you please change `return` to `returns`? Thanks.

File: spec/fixtures/syntax_highlight.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+6
+7
+</pre></td><td class="rouge-code"><pre><span style="color: #aa759f">module</span> <span style="color: #f4bf75">HelloWorld</span>
+  <span style="color: #aa759f">class</span> <span style="color: #f4bf75">Helo</span>
+    <span style="color: #aa759f">def</span> <span style="color: #303030">say</span><span style="color: #d0d0d0">(</span><span style="color: #303030">name</span><span style="color: #d0d0d0">)</span>
+      <span style="color: #90a959">"Hi, </span><span style="color: #8f5536">#{</span><span style="color: #303030">name</span><span style="color: #8f5536">}</span><span style="color: #90a959">!"</span>
+    <span style="color: #aa759f">end</span>
+  <span style="color: #aa759f">end</span>
+<span style="color: #aa759f">end</span>
+</pre></td></tr></tbody></table>
```
Comment (lucascaton) [inline spec/fixtures/syntax_highlight.txt]: Should we rename this file extention to `*.html`?

**Comment by coderhs** [comment]
@lucascaton done and updated the PR.

**Comment by lucascaton** [comment]
Awesome, thanks @coderhs!

**Comment by lucascaton** [comment]
@coderhs looks like there's a spec failing.
Each time we run specs, PDF gets a different [file hash](https://github.com/lucascaton/code2pdf/blob/master/spec/code2pdf/convert_to_pdf_spec.rb#L12), could you please fix it?

**Comment by coderhs** [comment]
Ya I saw that. Would. Need to change that test as it started happening
since I used pdftext guess its adding some dynamic value in it.

I will use something like pdf inspector and confirm the contents of the
file. Instead of looking at the hash.

**Comment by lucascaton** [comment]
Sounds awesome, thanks @coderhs.

**Comment by coderhs** [comment]
@lucascaton https://github.com/lucascaton/code2pdf/pull/7

**Comment by lucascaton** [comment]
@coderhs thanks!
Merged and released: https://rubygems.org/gems/code2pdf/versions/0.4.1

**Pull request merged**